src/main/java/com/direwolf20/buildinggadgets/common/items/gadgets/GadgetBuilding.java

```diff
@@ -141,9 +140,6 @@ public void addInformation(ItemStack stack, @Nullable World world, List<ITextCom
         player.setActiveHand(hand);
         if (!world.isRemote) {
             if (player.isSneaking()) {
-                //TODO Remove debug code
-                IEnergyStorage energy = EnergyUtil.getCap(itemstack).orElseThrow(CapabilityNotPresentException::new);
-                int accepted = energy.receiveEnergy(105000, false);
```

Comment:
Don't remove our Debug code! (The removal comment is for us, so that we find it again when we are finished with the most parts of debugging 1.13)